src/lib/FieldType/Query/Type.php

```diff
@@ -237,6 +239,13 @@ public function validateFieldSettings($fieldSettings)
 
         return $errors;
     }
+
+    public static function getTranslationMessages(): array
```

Comment:
```suggestion
    /**
     * @return string[]
     */
    public static function getTranslationMessages(): array
```

Reply:
Docblock is inherited from an interface `TranslationContainerInterface` and it is 
```
    /**
     * @return array<Message>
     */
```
There is no need to add it